src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -164,201 +163,181 @@ int main( int argc, char** argv )
+      fespace.GetEssentialVDofs( ess_bdr, new_ess_vdof_marker, d );
+      // Compute union of existing marked vdofs with vdofs marked on dimension d
+      for ( int j = 0; j < new_ess_vdof_marker.Size(); ++j ) {
+        ess_vdof_marker[j] = ess_vdof_marker[j] || new_ess_vdof_marker[j];
+      }
     }
     // Convert the vdofs to tdofs to remove duplicate values over ranks
     mfem::Array<int> ess_tdof_marker;
-    par_fe_space.GetRestrictionMatrix()->BooleanMult( ess_vdof_marker, ess_tdof_marker );
+    fespace.GetRestrictionMatrix()->BooleanMult( ess_vdof_marker, ess_tdof_marker );
     // Convert the tdof marker array to a tdof list
     mfem::FiniteElementSpace::MarkerToList( ess_tdof_marker, ess_tdof_list );
   }
   timer.stop();
   SLIC_INFO_ROOT( axom::fmt::format( "Time to set up boundary conditions: {0:f}ms", timer.elapsedTimeInMilliSec() ) );
 
-  // This block of code constructs a small-deformation linear elastic bilinear
-  // form.
+  // This block of code constructs a small-deformation linear elastic bilinear form.
   timer.start();
-  mfem::ParBilinearForm a( &par_fe_space );
+  mfem::ParBilinearForm a( &fespace );
   mfem::ConstantCoefficient lambda_coeff( lambda );
   mfem::ConstantCoefficient mu_coeff( mu );
   a.AddDomainIntegrator( new mfem::ElasticityIntegrator( lambda_coeff, mu_coeff ) );
 
   // Assemble the on-rank bilinear form stiffness matrix.
   a.Assemble();
-  // Reduce to tdofs and form a hypre parallel matrix for parallel solution of
-  // the linear system.
-  auto A = std::make_unique<mfem::HypreParMatrix>();
-  a.FormSystemMatrix( ess_tdof_list, *A );
+  // Reduce to tdofs and form a hypre parallel matrix for parallel solution of the linear system.
+  auto A_elasticity = std::make_unique<mfem::HypreParMatrix>();
+  a.FormSystemMatrix( ess_tdof_list, *A_elasticity );
   timer.stop();
   SLIC_INFO_ROOT(
       axom::fmt::format( "Time to create and assemble internal stiffness: {0:f}ms", timer.elapsedTimeInMilliSec() ) );
 
   // This block of code does initial setup of Tribol.
   timer.start();
-  // Next, we create a Tribol coupling scheme between the contact surfaces on
-  // the MFEM mesh. To create the coupling scheme requires several steps: 1)
-  // building a boundary submesh, 2) building a LOR mesh (if required), 3)
-  // re-decomposing the domain to move spatially close surface element pairs on
-  // to the same rank, 4) creating Tribol meshes of each surface, and 5)
-  // registering the meshes and coupling scheme with Tribol. These 5 steps are
-  // performed by calling two methods: 1) registerMfemCouplingScheme() (steps 1
-  // and 2) and 2) updateMfemParallelDecomposition() (steps 3, 4, and 5).
-  // registerMfemCouplingScheme() is called here and
-  // updateMfemParallelDecomposition() is typically called before calling
-  // update().
+  // Next, we create a Tribol coupling scheme between the contact surfaces on the MFEM mesh. To create the coupling
```

Comment:
Similar minor comment about being explicit about what is a parallel object vs. serial object.